package.json

```diff
@@ -48,18 +48,18 @@
         "@11ty/eleventy": "^2.0.1",
         "@11ty/eleventy-navigation": "^0.3.5",
         "@11ty/eleventy-plugin-syntaxhighlight": "^4.2.0",
-        "@patternslib/dev": "^3.3.5",
+        "@patternslib/dev": "^3.4.0",
         "@types/sinon": "^10.0.14",
         "npm-run-all": "^4.1.5",
         "rimraf": "^4.1.3",
-        "sinon": "^15.0.4"
+        "sinon": "^15.2.0"
```

Comment:
Turned out, that this sinon version has to be in `transformIgnorePattern` in jest config. Now tests are green again.